client/src/GameState.ts

```diff
@@ -53,12 +54,11 @@ export class GameState {
             console.log("playerId: ", playerId);
         });
 
```

Comment:
Now that we are rotating the other players' tetrominoes when they come in, the initial state needs to be rotated as well. (so that all players appear on the four sides instead of collapsing to the top). I tried something like this in line 55 and it worked:
```ts
        // rotate other pieces to their initial position
        this.otherPieces.map((state, i) => {
            state.setRotatedPosition(state.position, i + 1);
            state.setRotation(i + 1 + state.rotation);
        })

```

Reply:
Yeah, so the way it works right now is that we initialize the rendered tetrominoes (both the one of the player and the three others) just in the standard top spawn position, but then upon the first fall, everyone sends their tetromino state updates and the drawn tetrominoes are updated to the correct position.

I thought about addressing this "before we receive an update from the first fall" position, but skipped it because I think we are going to have to modify that code anyways. We _do_ know the starting position and rotation for the other players, however we _don't_ know the tetromino type, so we probably need all clients to send a tetromino state update on-initialization (before the first fall), and have the clients wait to receive the update from their peers before initializing and drawing their peers' tetrominoes.